src/Umbraco.Core/CoreBootManager.cs

```diff
@@ -455,14 +457,17 @@ protected virtual void InitializeResolvers()
             {
                 ServerRegistrarResolver.Current = new ServerRegistrarResolver(new ConfigServerRegistrar());
             }
+            else if ("true".InvariantEquals(ConfigurationManager.AppSettings["umbracoDisableElectionForSingleServer"]))

```

Comment:
Don't think so, it's going to be `null` if missing, and `"true".InvariantEquals(null)` is false... I think it's the reason why I'm doing `"true".InvariantEquals` and not the opposite (appsetting invariant equals true).